app/main.py

```diff
@@ -1,6 +1,20 @@
+import os
+import requests
+from dotenv import load_dotenv
+load_dotenv()
```

Comment:
Separate build in and third-party imports with 1 blank line. Add 2 blank line after imports